addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6263,7 +6263,7 @@ msgstr ""
 #. Description of setting "Videos -> Playback -> Use VC-1 VDPAU" with label #13445
 #: system/settings/settings.xml
 msgctxt "#13446"
-msgid "Enable this option to use hardware acceleration for VC-1 based codecs. If disabled the CPU will be used instead. AMD Hardware with VDPAU cannot decode VC-1 Simple."
+msgid "Enable this option to use hardware acceleration for VC-1 based codecs. If disabled the CPU will be used instead. AMD hardware with VDPAU cannot decode VC-1 simple."
```

Comment:
The last sentence reads odd. @fritsch is "VC-1 Simple" referring to a certain VC-1 standard/level or should the last sentence mean that AMD hardware can't decode VC-1 with VDPAU?